tip-payment/scripts/autosnapshot.sh

```diff
@@ -122,8 +122,8 @@ create_snapshot_for_slot() {
   # for some reason solana-ledger-tool error doesn't cause this script to exit out, so check status here
   exit_status=$?
   if [ $exit_status -ne 0 ]; then
-      echo "solana-ledger-tool returned $exit_status"
-      exit $exit_status
+    echo "solana-ledger-tool returned $exit_status"
```

Comment:
does this check not get executed due to `set -eux` on line 9?

Reply:
eg 

```
#!/bin/bash

set -exu
function foo {
  rm aldskjdjkldlkjadskjldkjlskjlkjdlslkjda
  echo $?
}

foo

echo 'finished'
```

Reply:
```
➜  ~ bash demo.sh
+ foo
+ rm aldskjdjkldlkjadskjldkjlskjlkjdlslkjda
rm: cannot remove 'aldskjdjkldlkjadskjldkjlskjlkjdlslkjda': No such file or directory
➜  ~ echo $?
1
```
the `echo finished` never runs

Reply:
yeah im not totally sure why it doesn't work, there's a comment above it.